proprietary/tokens/src/components/ams/grid.tokens.json

```diff
@@ -2,17 +2,29 @@
   "ams": {
     "grid": {
       "column-count": { "value": "4" },
-      "column-gap": { "value": "{ams.space.grid.md}" },
+      "column-gap": { "value": "{ams.space.xl}" },
       "padding-block": {
-        "sm": { "value": "{ams.space.grid.sm}" },
-        "md": { "value": "{ams.space.grid.md}" },
-        "lg": { "value": "{ams.space.grid.lg}" }
+        "sm": {
+          "comment": "Grows from ams.space.md to ams.space.lg.",
+          "value": "clamp(1.125rem, 0.8438rem + 1.4063vw, 2.25rem)"
+        },
+        "md": {
+          "comment": "Grows from ams.space.lg to ams.space.xl.",
+          "value": "clamp(1.6875rem, 1.2344rem + 2.2656vw, 3.5rem)"
+        },
+        "lg": {
+          "comment": "Grows from ams.space.xl to ams.space.2xl.",
+          "value": "clamp(2.25rem, 1.5rem + 3.75vw, 5.25rem)"
+        }
+      },
+      "padding-inline": {
+        "comment": "Grows from ams.space.lg to ams.space.2xl.",
+        "value": "clamp(1.6875rem, 0.7969rem + 4.4531vw, 5.25rem)"
```

Comment:
I still don't really get why we choose to diverge from our spacing system here. If you look at a screen on 832px, you'll see spaces of 20.4px (md), 30.6px (lg) and 44px (xl) everywhere. Except for our grid, which has 38.6px block padding and 49.8px inline. Why do we choose to add this inconsistency? It makes the spacing system less of a complete system, imo.